src/components/FullWidthPromo/FullWidthPromo.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text } from '@storybook/addon-knobs';
+
+import FullWidthPromo from './FullWidthPromo';
+
+const FullWidthPromoExample = () => {
+  const generalID = 'General';
+  const imageID = 'Image';
+  const metaID = 'Meta';
+
```

Comment:
I like this idea. Think we should adopt on all stories going forwards

Reply:
yes where it makes sense to i.e. loads of props